python/pyhail/dataset.py

```diff
@@ -553,6 +664,31 @@ def export_plink(self, output, fam_expr = 'id = s.id'):
     def export_samples(self, output, condition, types=None):
         """Export sample information to delimited text file.
 
+        **Examples**
+
+        Export some sample QC metrics:
+
+        >>> (hc.read('data/example.vds')
+        >>>   .sample_qc()
+        >>>   .export_samples('data/samples.tsv', 'SAMPLE = s, CALL_RATE = sq.qc.callRate, NHET = sa.qc.nHet'))
+
+        This will produce a file with a header and three columns.  To
+        produce a file with no header, just leave off the assignment
+        to the column identifier:
+
+        >>> (hc.read('data/example.vds')
+        >>>   .sample_qc()
+        >>>   .export_samples('data/samples.tsv', 's, CALL_RATE = sq.qc.rTiTv'))
+
+        **Notes**
+
+        One line per sample will be exported.  As :py:meth:`pyhail.VariantDataset.export_samples` runs in sample context, the following symbols are in scope:
```

Comment:
Add a tilde before pyhail. Should read :py:meth:\`~pyhail.VariantDataset.export_samples\`